packages/cf-component-icon/src/Icon.js

```diff
@@ -76,11 +288,18 @@ Icon.propTypes = {
     'widen',
     'wrench'
   ]).isRequired,
-  size: PropTypes.oneOf(['2x', '3x', '4x', 'large', 'xlarge']),
```

Comment:
So I was hoping we'd leverage the power of `fela` to set the size in the context of how the icon is used? To me that makes more sense so I thought I'd deprecate it